src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -116,7 +123,7 @@ bool FrontEnd::supported_impl(const std::vector<ov::Any>& variants) const {
 }
 
 ov::frontend::InputModel::Ptr FrontEnd::load_impl(const std::vector<ov::Any>& variants) const {
-    // TODO: Support other TensorFlow formats: SavedModel, .meta, checkpoint, pbtxt
+    // TODO: Support other TensorFlow formats: SavedModel, .meta, checkpoint
```

Comment:
I didn't see passed results with existing models to declare it is supported.

Could you share, please?

Reply:
Please take a look at TF FE unit-tests: https://dev.azure.com/openvinoci/dldt/_build/results?buildId=528226&view=logs&j=982848f2-7f8d-5e57-730a-8df1e5f991ba&t=c36556a9-9217-5d3d-3f01-b110be4c2de5
Now most of them moved to pbtxt

Also have other MO unit-tests also working with pbtxt file: https://dev.azure.com/openvinoci/dldt/_build/results?buildId=528226&view=logs&j=982848f2-7f8d-5e57-730a-8df1e5f991ba&t=cf9ee82e-e056-500f-81f4-f6a3a79d25db

We have more than 10 tests for pbtxt format